src/Chapter1.hs

```diff
@@ -632,7 +640,13 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n =
+  let
+    lastDigit2 = mod (abs n) 10
+    secondDigit = mod (div (abs n - lastDigit2) 10) 10
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

Note: For `div` the last digit won't matter, so it is okay not to subtract it in here 🙂 

One hint to make your solution even shorter: you see that you use both of these:

```haskell
mod (abs n) 10
div (abs n) 10 -- div (abs n - lastDigit2) 10
```

The standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(a, b) = divMod (abs n) 10
```

You can see how we could pattern match on the pair 🙂